release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -667,6 +667,146 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
+==== Monitoring components updated to use TLS authentication for metrics collection
+
+With this release, all monitoring components are now configured to use mutual TLS authentication, rather than Bearer Token static authentication for metrics collection. TLS authentication is more resilient to Kubernetes API outages and decreases the load on the Kubernetes API.
+
+[id="ocp-4-10-monitoring-updated-cmo-to-use-global-tls-profile"]
+==== Cluster Monitoring Operator updated to use the global TLS security profile
+
+With this release, the Cluster Monitoring Operator components now honor the global `tlsSecurityProfile` settings. 
+The following components and services now use the TLS security profile:
+
+* Alertmanager pods (ports 9092 and 9097).
+* `kube-state-metrics` pod (ports 8443 and 9443).
+* `openshift-state-metrics` pod (ports 8443 and 9443).
+* `node-exporter` pods (port 9100).
+* Grafana pod (port 3002).
+* `prometheus-adapter` pods (port 6443).
+* `prometheus-k8s` pods (ports 9092 and 10902).
+* Thanos query pods (ports 9092, 9093 and 9094).
+* Prometheus Operator (ports 8080 and 8443).
+* `telemeter-client` pod (port 8443)
+
+If you have enabled user-defined monitoring, the following pods now use the profile:
+
+* `prometheus-user-workload` pods (ports 9091 and 10902).
+* `prometheus-operator pod` (ports 8080 and 8443).
+
+[id="ocp-4-10-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+* *New*
+** Added a `namespace` label to all Thanos alerting rules.
+** Added the `openshift_io_alert_source=”platform”` label to all platform alerts. 
+
+
+* *Changed*
+** Renamed `AggregatedAPIDown` to `KubeAggregatedAPIDown`.
+** Renamed `AggregatedAPIErrors` to `KubeAggregatedAPIErrors`.
+** Removed the `HighlyAvailableWorkloadIncorrectlySpread` alert. This alert is no longer needed because the Cluster Monitoring Operator ensures hard anti-affinity for all monitoring components.
+** Improved the description of the `KubeMemoryOvercommit` alert.
+** Excluded `ReadOnlyMany` volumes from the `KubePersistentVolumeFillingUp` alerts.
+** Improved `NodeFilesystemSpaceFillingUp` alerts to make it consistent with the Kubernetes garbage collection thresholds.
+** Extended `PrometheusOperator` alerts to include the Prometheus operator running in the `openshift-user-workload-monitoring` namespace.
+** Replaced the `ThanosSidecarPrometheusDown` and `ThanosSidecarUnhealthy` alerts by `ThanosSidecarNoConnectionToStartedPrometheus`.
+** Changed the severity of `KubeletTooManyPods` from `warning` to `info`.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+=====
+
+[id="ocp-4-10-monitoring-changes-to-metrics"]
+==== Changes to metrics
+
+* Pod-centric cAdvisor metrics available at the slice level have been dropped.
+* The following metrics are now exposed:
+** `kube_poddisruptionbudget_labels`
+** `kube_persistentvolumeclaim_labels`
+** `kube_persistentvolume_labels metric`
```

Comment:
`kube_persistentvolume_labels metric`
remove metric

Reply:
Removed this metric from the list